bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -168,6 +174,8 @@ def __init__(
         enforce_dtype: bool = False,
         shape: t.Optional[t.Tuple[int, ...]] = None,
         enforce_shape: bool = False,
+        from_parquet: bool = False,
+        to_parquet: bool = False,
```

Comment:
just a thought, maybe instead of having `from_parquet` and `to_parquet`, lets just do `parquet: bool=True|False`? It seems to me that these two fields are similar, so instead of having duplicates field we can just have one?

Reply:
also should we make it a behaviour to `enforce_parquet`?

Reply:
I have a similar feedback here, since `PandasDataFrame` is design to be used for both input and output of an API endpoint, a `to_parquet` parameter would not make sense when the instance is being passed to API's input parameter. Also considering we may want to add support for csv or arrow as additional wired format, I'd suggest to make it `format=json|parquet|csv`, that way it makes the API easier to extend in the future.

Reply:
Or we could make it `default_format=json|parquet|csv`, indicating the default wire format to use. And we can decide how to parse the client request dynamically, based on the content-type header. That way, one API endpoint can support both JSON and parquet simultaneously, it's just that client side need to specify a header, indicating the format. What do you think?

Reply:
I think having the options to pass the wire format from content-type header is very powerful. `default_format` seems the nicest of all.

Reply:
Good suggestions! I've given the `default_format` a shot. Let me know what you think.